src/components/ModelingPageProvider.tsx

```diff
@@ -0,0 +1,175 @@
+import React, { useEffect, useMemo } from 'react'
```

Comment:
This was the `fileMachineProvider.tsx` but now it is called `ModelingPageProvider` because it has some initialization for the modeling page. No more file machine related stuff.

This will most likely get removed in the future when more command logic gets refactored.